src/aap_eda/core/migrations/0022_activation_organization_auditaction_organization_and_more.py

```diff
@@ -0,0 +1,139 @@
+            model_name="credential",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="decisionenvironment",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="extravar",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="project",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="role",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="rulebook",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="rulebookprocess",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="rulebookprocesslog",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="organizations",
+            field=models.ManyToManyField(
+                help_text="The list of organizations the user is in.",
+                related_name="users",
+                to="core.organization",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="team",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="core.organization",
+            ),
+        ),
+    ]
```

Comment:
The `0021_organization_team` migration is the last one that runs before DAB RBAC migrations. I think it makes it the easiest for us to put these kind of changes in that migration file or before. This means that there is no possibility that DAB RBAC will initialize without these models, and we don't need to worry any corner cases that might result from that. Just adding these steps to that file seems like the simplest thing to do right now.